teps/0096-pipelines-v1-api.md

```diff
@@ -174,11 +182,106 @@ V1 work should prioritize stabilizing existing beta features that support the [u
+    - We will proceed with [TEP-0074: Deprecate PipelineResources](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md)
+    independently of Pipelines V1, and merging this TEP as implementable isn't a V1 blocker.
+
+#### Behavior flags
+
+Existing feature flags can be found [here](https://github.com/tektoncd/pipeline/blob/main/config/config-feature-flags.yaml).
+The feature flags "disable-home-env-overwrite", "disable-working-directory-overwrite", and "scope-when-expressions-to-task"
+are not present in this table, as their deprecation has already [been announced](https://github.com/tektoncd/pipeline/blob/main/docs/deprecations.md#deprecation-table).
+
+| Flag                                          | Current Default | Proposed State                                       |
+| --------------------------------------------- | --------------- | -----------------------------------------------------|
+| disable-affinity-assistant                    | false           | default to true                                      |
+| disable-creds-init                            | false           | no change                                            |
+| running-in-environment-with-injected-sidecars | true            | no change                                            |
+| require-git-ssh-secret-known-hosts            | false           | no change                                            |
+| enable-tekton-oci-bundles                     | false           | collapsed under enable-api-fields                    |
+| enable-custom-tasks                           | false           | collapsed under enable-api-fields                    |
+| enable-api-fields                             | stable          | default to stable, "beta" option added               |
+
+### Feature Completeness
+
+No features identified as V1 blockers that are not necessary for stability or production readiness.
 
 ## Future Work (Out of Scope for V1)
 
+- Supply chain security features
+- Support for manual approval or pause and resume in Pipelines
+  - This would be supported by the [wait CustomTask](https://github.com/tektoncd/experimental/tree/main/wait-task) or another experimental CustomTask.
+- Native notification support, as proposed in [TEP-0032: Notifications](https://github.com/tektoncd/community/blob/main/teps/0032-tekton-notifications.md)
+- Dynamically generated TaskRuns within a PipelineRun, as proposed in [TEP-0090: Matrix](https://github.com/tektoncd/community/blob/main/teps/0090-matrix.md)
+- Stabilizing the Run CRD
```

Comment:
This means from beta to stable, right?

Reply:
yup! updated the text to make this clear